src/useCases/common/shareFile.ts

```diff
@@ -0,0 +1,21 @@
+import { DisplayableError } from '@internxt-mobile/services/common';
+
+import fileSystemService from '@internxt-mobile/services/FileSystemService';
+import { FileSystemRef } from '@internxt-mobile/types/index';
+import strings from 'assets/lang/strings';
+
+export const shareFile = async ({ title, filePath }: { title: string; filePath: FileSystemRef }) => {
```

Comment:
`async` is not required here, I would say that is better to just define the `return` type of the function, even if it appears now obvious, as in the future someone could change it.